src/Runner.Worker/ActionManager.cs

```diff
@@ -399,6 +399,13 @@ public Definition LoadAction(IExecutionContext executionContext, Pipelines.Actio
                     else if (definition.Data.Execution.ExecutionType == ActionExecutionType.NodeJS)
                     {
                         var nodeAction = definition.Data.Execution as NodeJSActionExecutionData;
+                        
+                        string nodeVersionOut = Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedActionsNodeVersion);
```

Comment:
sorry... my bad...
i was suggesting `NodeScriptActionHandler` which we already have some code to force to use node16 for osx-arm64.